src/pages/resultsView/mutation/MutationMapper.tsx

```diff
@@ -84,10 +84,8 @@ export default class MutationMapper extends React.Component<IMutationMapperProps
                     hugoGeneSymbol={this.props.store.gene.result.hugoGeneSymbol}
                     mutations={this.props.store.mutationData.result}
                     sampleIds={this.props.store.sampleIds.result}
-                    patientIds={this.props.store.patientIds.result}
-                    patientIdsStatus={this.props.store.patientIds.status}
-                    mskImpactGermlineConsentedPatientIds={this.props.store.mskImpactGermlineConsentedPatientIds.result}
-                    mskImpactGermlineConsentedPatientIdsStatus={this.props.store.mskImpactGermlineConsentedPatientIds.status}
+                    germlineConsentedSampleIds={this.props.store.germlineConsentedSampleIds.result}
```

Comment:
you could type it as an interface satisfying { result: T, status: ...}, which is a type we could use elsewhere too